Circuit/Components/VacuumTubes/Pentode.cs

```diff
@@ -0,0 +1,113 @@
+﻿using ComputerAlgebra;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace Circuit.Components
+{
+    [Category("Vacuum Tubes")]
+    [DisplayName("Pentode")]
+    public class Pentode : Component
+    {
+        private Terminal _plate, _grid, _grid2, _cathode;
+
+        private double _mu = 10.7;
+        [Serialize, Category("Koren"), Description("Voltage gain.")]
+        public double Mu { get { return _mu; } set { _mu = value; NotifyChanged(nameof(Mu)); } }
+
+        private double _kg1 = 1672.0;
+        [Serialize, Category("Koren"), Browsable(true)]
+        public double Kg1 { get { return _kg1; } set { _kg1 = value; NotifyChanged(nameof(Kg1)); } }
+
+        private double _kg2 = 4500;
+        [Serialize, Category("Koren"), Browsable(true)]
+        public double Kg2 { get { return _kg2; } set { _kg2 = value; NotifyChanged(nameof(Kg2)); } }
+
+        private double _kp = 41.16;
+        [Serialize, Category("Koren"), Browsable(true)]
+        public double Kp { get { return _kp; } set { _kp = value; NotifyChanged(nameof(Kp)); } }
+
+        private double _kvb = 12.7;
+        [Serialize, Category("Koren"), Browsable(true)]
+        public double Kvb { get { return _kvb; } set { _kvb = value; NotifyChanged(nameof(Kvb)); } }
+
+        private double _ex = 1.310;
+        [Serialize, Category("Koren"), Browsable(true)]
+        public double Ex { get { return _ex; } set { _ex = value; NotifyChanged(nameof(Ex)); } }
+
+        public Pentode()
```

Comment:
Isn't this a Tetrode, not a Pentode?

Reply:
Technically yes. Because in typical applications suppressor grid is connected with cathode (some of the tubes have them connected internally) Koren uses single model for tetrodes and pentodes, so maybe class naming isn't exactly correct ;)